source/2019-04-28-the-ember-times-issue-96.md

```diff
@@ -10,91 +10,91 @@ responsive: true
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
-  
+This week:
+- **easier navigation through concurrency** 🕗 in your app tests,
+- Ember **contributions** from the **EmberJS Paris** contributor workshop 🇫🇷,
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Merci for Your Contributions at EmberJS Paris 🐹🇫🇷](https://twitter.com/EmberJSParis/status/1120760348485799939)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Last week's **contributor workshop** organised by the folks over at the [EmberJS Paris meetup](https://twitter.com/EmberJSParis) has been a **real success**: with new changes to Ember CLI, improvements to Ember addons and even a brand-new RFC written, the Ember community gave back to the project, with some of the work even being made by first-time contributors! 🙌
+
+Already feeling inspired to participate in or initiate your own **Ember contributor workshop**? Be sure to check out [these helpful resources to help you get you started](https://github.com/ember-learn/emberjs-contributors-workshop). And don't forget to check back on [\#dev-ember-learning Discord channel](https://discordapp.com/invite/emberjs) and the [Ember Help Wanted app](https://help-wanted.emberjs.com/) to find a something neat to work on. Codage heurex! 💻❤️
```

Comment:
I don't know French, sadly. Google said the spelling is `heureux`, and I wasn't sure if `heurex` is the same.

Reply:
That's right, it should be `heureux` - thank you for the double check 🇫🇷